src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs

```diff
@@ -246,6 +246,23 @@ private static void VerifyBytes(IntPtr p, int length)
         }
 
         [Fact]
+        public static void ValidateExceptionForHRAPI()
+        {
+            int errorCode = -2146231029;
+            COMException getHRException = Marshal.GetExceptionForHR(errorCode) as COMException;
+            Assert.NotNull(getHRException);
+            try
+            {
+                Marshal.ThrowExceptionForHR(errorCode);
+            }
+            catch (COMException e)
+            {             
+                Assert.Equal(e.HResult, errorCode);
```

Comment:
This is backwards, expected is first, so 
`Assert.Equal(errorCode, e.HResult);`